api/pom.xml

```diff
@@ -31,6 +31,14 @@
   </build>
 
   <dependencies>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
```

Comment:
Can you order the dependencies alphabetically by `groupId`? So these would be below `io.dropwizard`